src/lib/routes/admin-api/project/api-token.ts

```diff
@@ -98,9 +103,13 @@ export class ProjectApiTokenController extends Controller {
                     tags: ['Projects'],
                     operationId: 'createProjectApiToken',
                     requestBody: createRequestSchema('createApiTokenSchema'),
+                    summary: 'Create a project API token.',
+                    description:
+                        'Endpoint that allows creation of [project API tokens](https://docs.getunleash.io/how-to/how-to-create-project-api-tokens) for the specified project.',
                     responses: {
                         201: resourceCreatedResponseSchema('apiTokenSchema'),
                         400: emptyResponse,
+                        ...getStandardResponses(401, 403),
```

Comment:
This also returns a 400, so I'll add that

Reply:
Never mind, it already had 400

Reply:
Can we add 400 as part of the standard response? Maybe?

Reply:
I don't see why not